SSO-Auth/Config/configPage.html

```diff
@@ -4,12 +4,177 @@
     <title>SSO</title>
```

Comment:
Lots of extra (unneeded) newlines here too

Reply:
Ran a formatter against this, as well as the js